test/e2e/bundle_e2e_test.go

```diff
@@ -165,6 +165,88 @@ var _ = Describe("Installing bundles with new object types", func() {
 			}).Should(Succeed())
 		})
 	})
+
+	When("A bundle is installed with a CR that is associated with a provided CRD", func() {
+		By("including the VPA CRD in the CSV")
+		const (
+			packageName = "busybox"
+			channelName = "alpha"
+			subName     = "test-subscription"
+			vpaName     = "busybox-vpa"
+		)
+
+		BeforeEach(func() {
+			const (
+				sourceName = "test-catalog"
+				imageName  = "quay.io/olmtest/single-bundle-index:vpa-race"
```

Comment:
The test looks good to me, but it's not ideal to take a another dependency on an external catalog image.

Reply:
Agreed -- but unfortunately I don't see another option until our bundle building in tests effort merges